pacsman/dcmtk_client.py

```diff
@@ -64,6 +64,7 @@ def __init__(
         pacs_url,
         pacs_port,
         dicom_dir,
+        dicom_tmp_dir,
```

Comment:
Could you please add a line to the docstring for `dicom_tmp_dir`, below, and also convert `dicom_tmp_dir` to a keyword argument that defaults to `None`? This will ensure it remains backwards-compatible with previous versions of `pacsman`.

Reply:
Yes I actually meant to do this and left it out after forking. Sorry about that! Ill add the docstring too.